packages/app/src/components/VisualizationOptions/VisualizationOptionsManager.js

```diff
@@ -1,18 +1,23 @@
+import {
+    AXIS_ID_COLUMNS,
+    hasCustomAxes,
+    hasRelativeItems,
+    isDualAxisType,
+    VisualizationOptions,
+} from '@dhis2/analytics'
 import i18n from '@dhis2/d2-i18n'
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { getOptionsByType } from '../../modules/options/config'
 import {
-    ButtonStrip,
-    Modal,
-    ModalTitle,
-    ModalContent,
-    ModalActions,
-} from '@dhis2/ui'
-import React, { Component, Fragment } from 'react'
-import HideButton from '../HideButton/HideButton'
+    sGetUiType,
+    sGetUiOptions,
+    sGetDimensionItemsByAxis,
+    sGetUiLayout,
+} from '../../reducers/ui'
 import MenuButton from '../MenuButton/MenuButton'
-import UpdateButton from '../UpdateButton/UpdateButton'
 import UpdateVisualizationContainer from '../UpdateButton/UpdateVisualizationContainer'
-import styles from './styles/VisualizationOptionsManager.module.css'
-import VisualizationOptions from './VisualizationOptions'
 
 class VisualizationOptionsManager extends Component {
```

Comment:
Perhaps for another PR but this also can be converted into a functional component with hooks.